tests/TestRun2/UserFiles/UserCreds_10.csv

```diff
@@ -1,11 +1,101 @@
-username,password,name
-lis8fhithvx6s0pjpp,dBF@7_ThrhyW1f1VT8tYbczG@VdCb,Sabrina Murphy
-s2v8g00zbmkqbu,EpUe8@3PyG9U_y0fuLEk,Amber Sullivan
-fejtk07w2f8r5k9y2b4l,iG2OfqTIy9lZA_pGFHdU,Reginald Richards
-zv6d53sdzqzz8pq9pzui,lN9k_4Wooe1YczdHV0ldpD78,Benjamin Perez
-0jweqdqs7t87,u_C9YvXnT52,Amanda Whitehead
-3yqiaj0fd6x87ck3p72,9jiZho31t_8EWaA9Uj964D4l6,Janice Marshall
-nub390xf62m85,ErS4mbvxPg3FS9Cnd_ZeJadqIB8l,Jason Foster
-wkr2f3cok94lkvl,rJR56y@nI7L3ytP,Adam Michael
-8rr5ojjyrr527zz9b9,MquD1CF3@Z3ymD_XWM7sNsuldI,Tina Riggs
-ewurd9bqn1m,wBW26K1IS6@HtCYL5sM3UuLTK,Tyler Miller
+Username,Password,Name

```

Comment:
We shouldn't change this file, we should make new test files for 100, 1000, 10000